frontend/control-center/src/components/TopBar/index.tsx

```diff
@@ -17,7 +17,8 @@ import {env} from '../../env';
 import {useAnimation} from 'render';
 import {useTranslation} from 'react-i18next';
 import i18next from 'i18next';
-import {ConfigServices, Language} from 'model/Config';
+import {Language} from 'model/Config';
+import {ComponentName} from 'model';
 
```

Comment:
(nitpick) we could just have one import line: 

`import {Language, ComponentName } from 'model';`